deploy/deployment_dev_cluster.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: registration-service-v0.1
+  namespace: toolchain-registration-service
+spec:
+  lookupPolicy:
+    local: false
+  tags:
+  - from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/codeready-toolchain/registration-service-v0.1:registration-service
+    importPolicy:
+      scheduled: true
+    name: registration-service
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: registration-service
+  name: registration-service
+  namespace: toolchain-registration-service
+spec:
+  replicas: 3
+  revisionHistoryLimit: 10
+  selector:
+    app: registration-service
+    deploymentconfig: registration-service
+  strategy:
+    activeDeadlineSeconds: 21600
+    resources: {}
+    rollingParams:
+      intervalSeconds: 1
+      maxSurge: 25%
+      maxUnavailable: 25%
+      timeoutSeconds: 600
+      updatePeriodSeconds: 1
+    type: Rolling
+  template:
+    metadata:
+      labels:
+        app: registration-service
+        deploymentconfig: registration-service
+    spec:
+      serviceAccountName: registration-service
+      containers:
+      - name: registration-service
+        image: ''
+        command:
+        - registration-service
+        imagePullPolicy: IfNotPresent
```

Comment:
I think, we should use `Always` as it's running in the dev cluster, which is production for us

Reply:
Done in 8af6a1af6a8d69f5871dffae62436a7dd8b011f2